conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -30,8 +30,9 @@ types:
         alias: string
       ErrorCode:
         values:
-          - PERMISSION_DENIED
           - INVALID_ARGUMENT
+          - UNAUTHENTICATED
+          - PERMISSION_DENIED
```

Comment:
This was a change to the IR - I think we now need to figure out our IR versioning story pronto before we cut any releases.